src/sonic-config-engine/minigraph.py

```diff
@@ -419,6 +419,8 @@ def parse_meta(meta, hname):
     mgmt_routes = []
     erspan_dst = []
     deployment_id = None
+    k8s_master = None
```

Comment:
>None [](start = 17, length = 4)

Better to use empty string.

Reply:
Why. None seems to be more apt.

Reply:
It will finally written to JSON file or other consumers. To be a good component, output in the easiest format, so no assumpton on the consumer is so apt.

---
In reply to: [414879077](https://github.com/Azure/sonic-buildimage/pull/4474#discussion_r414879077) [](ancestors = 414879077)